### DISCUSSION TITLE
Copilot Chat in VSCode switches to 'Start 30 day free trial', but I am subscribed

### DISCUSSION THREAD

**Discussion by mroedder-d7** [community/community#106757] [Copilot Conversations]
### Select Topic Area

Bug

### Body

See title.
Status says ready.

<img width="723" alt="Screenshot 2024-02-08 at 13 04 30" src="https://github.com/community/community/assets/129860311/2c9cfb52-bd75-46ec-9c9c-d316b3eacfd2">

But chat won´t open
<img width="548" alt="Screenshot 2024-02-08 at 13 05 05" src="https://github.com/community/community/assets/129860311/a401017a-e16f-41d2-bf21-0ee450a4c481">

Code completion and everything else works.

Diagnostics:
## Copilot

- Version: 1.159.0
- Build: prod
- Editor: vscode/1.86.0

## Environment

- http_proxy: n/a
- https_proxy: n/a
- no_proxy: n/a
- SSL_CERT_FILE: n/a
- SSL_CERT_DIR: n/a
- OPENSSL_CONF: n/a

## Feature Flags

- Custom Certificates: disabled
- Send Restricted Telemetry: disabled
- Chat: enabled

## Node setup

- Number of root certificates: 137
- Operating system: Darwin
- Operating system version: 22.6.0
- Operating system architecture: x64
- NODE_OPTIONS: n/a
- NODE_EXTRA_CA_CERTS: n/a
- NODE_TLS_REJECT_UNAUTHORIZED: n/a
- tls default min version: TLSv1.2
- tls default max version: TLSv1.3

## Network Configuration

- Proxy host: n/a
- Proxy port: n/a
- Kerberos SPN: n/a
- Reject unauthorized: disabled
- Fetcher: HelixFetcher

## Reachability

- github.com: HTTP 200 - 
- copilot-proxy.githubusercontent.com: HTTP 200 - 
- api.githubcopilot.com: HTTP 200 - 
- default.exp-tas.com: HTTP 200 - 

## VS Code Configuration

- HTTP proxy: 
- HTTP proxy authentication: n/a
- Proxy Strict SSL: true
- Extension HTTP proxy support: override

## Extensions

- Is `win-ca` installed?: false
- Is `mac-ca` installed?: false

## Authentication

- GitHub username: mroedder-d7

**Comment by weskor** [comment]
I have the same issue.

**Reply by FloSch62** [reply]
same with me...

**Comment by leciric** [comment]
I have the same issue, it started after an update. 
On github it says "You have an active Copilot subscription." but in vscode I only have the start 30 day free trial button on chat.
Only code completions are still working, inline chat is also off

**Comment by huyz** [comment]
Can we have a GitHub employee acknowledge this important issue?

**Comment by aadium** [comment]
Same with me as well.

**Comment by syeed-mahmud** [comment]
Same with me.

**Comment by infatuation-psa** [comment]
Also happening on my end. I've tried removing it, adding it back and reinitializing, no dice.

**Comment by Meruman** [comment]
I have the same problem, I tried uninstalling the extensions, and also VSCode and installing it again with no success... I also tried going one version back from the github chat extension without success...

**Comment by hnfNaufal** [comment]
same issue right here

**Comment by kid-three** [comment]
Same issue

**Comment by Cow258** [comment]
![image](https://github.com/community/community/assets/14261588/ffa6d349-c850-45e9-81b9-4e3f865242ae)
Same issue

**Comment by Akash1134** [comment]
Hi All,

Sorry for the trouble caused. This issue is caused by VS Code not properly supporting multiple auth sessions. 

Ref:
https://github.com/microsoft/vscode-copilot-release/issues/857
https://github.com/microsoft/vscode/issues/203850
https://github.com/microsoft/vscode/issues/127967

Thanks!

**Reply by huyz** [reply]
I checked. Doesn't help:

VS Code: Version: 1.86.1
Copilot Chat: v0.13.2024020201 (pre-release)
also Copilot Chat: v0.12.1

**Reply by az-kennedy** [reply]
Signing out of one of my GitHub accounts in VSCode fixed this issue for me

**Reply by hnfNaufal** [reply]
this sign-out trick has fixed my issue as well, but this is still a problem for me because I need both of my accounts to work

**Reply by juniorbatistadev** [reply]
Still have the same issue after trying.


VS Code: Version: 1.86.1
Copilot Chat: v0.13.2024020201 (pre-release)
also tried Copilot Chat: v0.12.1

**Reply by huyz** [reply]
I had to log out of both my GitHub accounts and login again (probably in a very specific order) for it to work again

**Reply by Akash1134** [reply]
Hi everyone,

We are aware of this and have raised it to the concerned teams. As mentioned - this issue is caused by being logged into 2 GitHub accounts in VSCode which currently isn’t supported by VSCode.

References:

- https://github.com/microsoft/vscode/issues/203850
- https://github.com/microsoft/vscode/issues/127967

Thanks!

**Reply by pushkalkishore** [reply]
Please fix this issue. I need to be logged in with both my GitHub accounts.

**Comment by NPeredoEsquivel** [comment]
Same problem here

**Comment by rdzidziguri** [comment]
I have the same problem here. Signing out, and it did not help. In my case, I have no extension or VS code update available, as I am running the latest versions for both. If it was not a paid service fine, but Microsoft owns both products, taking money out of somebody's pocket and breaking things like that is unacceptable.

**Reply by juniorbatistadev** [reply]
Completely agree, they need to fix this ASAP and make sure they are careful not to break a paid service.

**Comment by rubn-g** [comment]
Same issue here, it's been 6 days already. Latest VSCode and GitHub Copilot versions, still failing. I can't sign out of my github accounts, i need both of them (work and personal)
We're paying for this product and the only message i've seen from you guys is requesting us to upgrade, where's the "sorry, we're working hard to fix it"?? That's a pretty bad customer service

**Reply by huyz** [reply]
>  I can't sign out of my github accounts, i need both of them (work and personal)

you only need to sign out temporarily and sign right back in.

**Reply by rdzidziguri** [reply]
that is not the case because I can not even find the option to sign in to second account anymore

**Reply by aadium** [reply]
Thanks. That fixed the issue

**Comment by SpookyJelly** [comment]
Well, at least I'm not the only one with this problem. It's very annoying.

**Comment by sephraim** [comment]
This happened to me after I updated VS Code to v1.86. I had to rollback to VS Code v1.85 to temporarily fix it until a more permanent fix becomes available.

Here is what I did...

**First turn off auto-updates for VS Code:**
1. Open VS Code.
2. Go to the _Code -> Settings_.
3. Search for "Update: Mode".
4. Set this to "none".

**Then rollback VS Code:**
1. Quit VS Code.
2. Download VS Code v1.85.2 from [here](https://code.visualstudio.com/updates/v1_85), and install it.
   - On Mac:
      1. Unzip the downloaded file.
      2. Copy the "Visual Studio Code" app into the Applications folder.
      3. When the prompt pops up, click "Replace".
3. Open VS Code.
4. Go to _Code -> About Visual Studio Code_, and make sure your current version is 1.85.2.
5. Uninstall the GitHub Copilot Chat extension.
6. Reinstall the GitHub Copilot Chat extension.
7. Restart VS Code.

Hopefully it should be working now.

You probably don't want to leave VS Code auto-updates turned off forever, so set a reminder to come back to this thread (in a few days? weeks? months? years???) and see if the issue has been fixed. Once it's fixed, turn auto-updates back on (set "Update: Mode" to "default").

Lastly, FWIW it appears that v0.12.0 of the GitHub Copilot extension is what introduced the "30-day free trial" error, and as of v0.12.2 it is still not fixed. Rolling back to VS Code v1.85 works because it only allows up to v0.11 of the extension, preventing v.0.12 from being installed in the first place. I tried running VS Code v1.86 with v0.11 of the extension but received different errors instead, so that's why I'm sticking with VS Code v1.85 for now. Hopefully it gets fixed soon.
